learn/advanced/tokenization.md

```diff
@@ -8,18 +8,18 @@ This allows Meilisearch to function in several different languages with zero set
 
 ## Deep dive: The Meilisearch tokenizer
 
-![Chart illustrating the architecture of Meilisearch's tokenizer](https://user-images.githubusercontent.com/6482087/102896344-8560d200-4466-11eb-8cfe-b4ae8741093b.jpg)
-
-When you add documents to a Meilisearch index, the tokenization process is handled by an abstract interface called an **analyzer**. The analyzer is responsible for determining the primary language of each field based on the scripts (e.g., Latin alphabet, Chinese hanzi, etc.) that are present there. Then, it applies the corresponding **pipeline** to each field.
+When you add documents to a Meilisearch index, the tokenization process is handled by an abstract interface called the tokenizer. The tokenizer is responsible for splitting each field by script (e.g., Latin alphabet, Chinese hanzi, etc.). It then applies the corresponding pipeline to each part of each field.
```

Comment:
Changing `script` to `writing system` since I believe it to be more accurate for some languages (for instance, Japanese has three different sets of characters/scripts). 

I also specified that the pipeline applies to each **document** field. Not sure it's necessary, but I figure it doesn't hurt.

```suggestion
When you add documents to a Meilisearch index, the tokenization process is handled by an abstract interface called the tokenizer. The tokenizer is responsible for splitting each field by writing system (e.g. Latin alphabet, Chinese hanzi). It then applies the corresponding pipeline to each part of each document field.
```